packages/forms/src/styled/file-list/StyledFile.ts

```diff
@@ -5,30 +5,104 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { rgba } from 'polished';
 import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { StyledFileClose } from './StyledFileClose';
 
 const COMPONENT_ID = 'forms.file';
 
+const colorStyles = (props: IStyledFileProps & ThemeProps<DefaultTheme>) => {
+  const borderColor = getColor('neutralHue', 300, props.theme);
+  const focusBorderColor = getColor('primaryHue', 600, props.theme);
+  const foregroundColor = getColor('neutralHue', 800, props.theme);
+  const boxShadow = `
+    ${props.focusInset ? 'inset' : ''}
+    ${props.theme.shadows.md(rgba(focusBorderColor!, 0.35))}`;
+
+  return `
+    border-color: ${borderColor};
+    color: ${foregroundColor};
+
+    &:focus {
+      box-shadow: ${boxShadow};
+      border-color: ${focusBorderColor};
+    }
+  `;
+};
+
+const sizeStyles = (props: IStyledFileProps & ThemeProps<DefaultTheme>) => {
```

Comment:
Separating out `colorStyles` and `sizeStyles` functions with related styling follows documented API [rules](https://github.com/zendeskgarden/react-components/blob/main/docs/api.md#rules) and is almost always desirable.